pallets/parachain-staking/src/weights.rs

```diff
@@ -253,20 +253,19 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().writes(7 as Weight))
 	}
 	fn round_transition_on_initialize(x: u32, y: u32) -> Weight {
-		(0 as Weight) // Standard Error: 4_087_000
-			// Standard Error: 12_000
-			.saturating_add((100_164_000 as Weight).saturating_mul(x as Weight))
-			.saturating_add((1_202_000 as Weight).saturating_mul(y as Weight))
-			.saturating_add(T::DbWeight::get().reads((4 as Weight).saturating_mul(x as Weight)))
-			.saturating_add(T::DbWeight::get().writes((3 as Weight).saturating_mul(x as Weight)))
+		(0 as Weight) // Standard Error: 2_467_000
+			.saturating_add((29_501_000 as Weight).saturating_mul(x as Weight))
+			// Standard Error: 9_000
```

Comment:
I'll update these with official benchmarks, should I go ahead and do that now?

Reply:
These are the official benchmarks, they were run on the target machine. I'm updating them now with the new run, it is slow.